app/src/main/AndroidManifest.xml

```diff
@@ -150,11 +150,13 @@
             android:exported="false"/>
 
         <!--suppress AndroidDomInspection -->
+<!--
         <provider
             android:name="com.github.mjdev.libaums.storageprovider.UsbDocumentProvider"
             android:exported="false"
             tools:node="remove">
         </provider>
+-->
```

Comment:
`UsbDocumentProvider` doesn't exist in the project.

Reply:
It does, and it will interfere with USB device grabbing. Please put it back.

Reply:
In any case this has no effect if the provider does not exist, there's no need to remove it.

Reply:
Try to find this class in the project with `double Shift`. 
But ok, it can stay as a manifest error